### ISSUE TITLE
calendar 1.6.0 current day not highlighted anymore?

### ISSUE THREAD

**Issue by himbeere** [nextcloud/calendar#726]
Hello Devs.

It seems with the release of calendar 1.6.0 the current day (meaning today) is not highlighted anymore. Until now "today" was light yellow or so to find "today" quickly. This is gone.  Can we have that back please? It makes the calendar more readable.

My system: NC 13, PHP 7.2.2.

thanks and cheers

**Comment by tcitworld** [comment]
More exactly the current day highlight is hidden behind the week-end highlighing.

**Comment by himbeere** [comment]
Good to know. So i guess this will be fixed soon. Partially tomorrow. :-)

thanks
t.

**Comment by ximex** [comment]
The problem is only that the css rules for saturday and sunday overrules the today rule.

**Comment by aditigupta17** [comment]
Can I take this issue up? Can I be redirected to the particular file which has to be modified?

**Comment by tcitworld** [comment]
Easy fix would be to have : 
```css
.fc-unthemed td.fc-today {
    background: #fcf8e3 !important;
}
```
at the bottom of calendarlist.scss for instance.

**Comment by georgehrke** [comment]
@aditigupta17 Do you want to send a pull-request for this? :) 
Can i assign you to this ticket?

**Comment by aditigupta17** [comment]
Ya sure! I am working on it currently, will be soon putting up the pull request.

**Comment by georgehrke** [comment]
fix is in #734

**Comment by georgehrke** [comment]
#734 was merged, fix will be included in Calendar 1.6.1

thx @aditigupta17
